src/snowflake/snowpark/dataframe.py

```diff
@@ -4188,6 +4189,117 @@ def _explain_string(self) -> str:
 
         return f"{msg}\n--------------------------------------------"
 
+    def map(
+        self,
+        func: Callable,
+        output_types: list[StructType],
+        output_column_names: Optional[list[str]] = None,
```

Comment:
I suggest adding a `*` before this parameter to make these optional parameters key-word arguments.

Reply:
Thanks for your suggestion.

I just removed the `extra_packages` parameter and added a `**kwargs` parameter to pass arguments to the UTDF registration (as with other similar methods).